2.4/root/usr/share/container-scripts/httpd/README.md

```diff
@@ -59,21 +59,30 @@ $ docker run -d --name httpd -p 8080:8080 httpd-app
 
 The structure of httpd-app can look like this:
 
-| Folder name       | Description                |
-|-------------------|----------------------------|
-| `./httpd-cfg`     | Can contain additional Apache configuration files (`*.conf`)|
-| `./httpd-pre-init`| Can contain shell scripts (`*.sh`) that are sourced before `httpd` is started|
-| `./httpd-ssl`     | Can contain own SSL certificate (in `certs/` subdirectory) and key (in `private/` subdirectory)|
-| `./`              | Application source code |
+**`Folder name`**  
```

Comment:
Is the original header supposed to be in here?

Reply:
It shouldn't, thanks for spotting this.

Reply:
Fixed now.